vue/pages/app/index.html

```diff
@@ -404,6 +404,19 @@ <h4><a href="/apps">APPS</a> > <span v-if="isNew()">{{ params.app.toUpperCase()
                               </div>
                             </div>
                         </div>
+                        <div v-if="app.type == 's3'" class="field is-horizontal">
+                          <div class="field-label is-normal">
+                            <label class="label">CloudFront (ID)</label>
+                          </div>
+                          <div class="field-body">
+                            <div class="field">
+                              <p class="control">
+                                <input class="input name" :disabled="!i.claims['edit'] && !user.admin" type="text"  v-model="i.cloudFrontId" title="The CloudFront Distribution ID that serves your static s3 bucket on a cdn.">
+                                <label class="help">Specify the CloudFront Distribution id that will be used to create an invalidation object upon deployment.</label>
```

Comment:
Maybe "Distribution id that is used to invalidate the cache after a deployment." Not sure you need to specify what it creates. If they have one, they should know this already. Let me know your thoughts.